ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-master-periodics.yaml

```diff
@@ -0,0 +1,37 @@
+periodics:
+- agent: kubernetes
+  cluster: api.ci
+  cron: '@hourly'
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: open-cluster-management
+    repo: registration-operator
+  name: periodic-open-cluster-management-registration-operator-fast-forward
+  spec:
+    containers:
+    - args:
+      - --config-dir=./ci-operator/config/open-cluster-management/registration-operator/
+      - --current-release=master
+      - --future-release=2.1
+      - --fast-forward
+      - --confirm
+      - --username=openshift-merge-robot
```

Comment:
It seems also that you are using different token. It doesn't make sense to use `openshift-merge-robot` as the username.

Reply:
updated.